pages/common/comm.md

```diff
@@ -0,0 +1,23 @@
+# comm
+
+> Select or reject lines common to two files. Both files must be sorted.
+
+- Produce three-tab separated columns: lines only in first file, lines only in second file and common lines.
```

Comment:
Are the columns separated by 3 tabs? or 3 columns separated by one tab each?

`A\t\t\tB` or `A\tB\tC`

Seems to be the first case when it should be the second.